exercises/difference-of-squares/canonical-data.json

```diff
@@ -22,6 +22,12 @@
           "property": "squareOfSum",
           "number": 100,
           "expected": 25502500
+        },
+        {
+          "description": "square of sum 2000",
```

Comment:
If this **were** to be added, a description that describes why the test is necessary should be used.
"result greater than 2^32" or something similar.

(Note consistency is not a good defence here as all the other tests are also poorly named.)